pkg/controller/template/render.go

```diff
@@ -33,20 +33,19 @@ const (
 )
 
 // generateMachineConfigs returns MachineConfig objects from the templateDir and a config object
-// expected directory structure for correctly templating machine configs: <templatedir>/<platform>/<role>/<type>/<tmpl_file>
+// expected directory structure for correctly templating machine configs: <templatedir>/<role>/<name>/<platform>/<type>/<tmpl_file>
 //
 // All files from platform _base are always included, and may be overridden or
 // supplemented by platform-specific templates
 //
 //  ex:
-//       templates/_base/worker/units/kubelet.conf.tmpl
-//                           /files/hostname.tmpl
-//                    /master/units/kubelet.tmpl
-//                           /files/hostname.tmpl
-//       templates/aws/worker/units/kubelet-dropin.conf.tmpl
+//       templates/worker/worker/_base/units/kubelet.conf.tmpl
```

Comment:
nit: It took me a second to realize the `worker/worker/` part was `<role>/<name/`. It may be worth adding one more example where they don't match. Not a big deal at all though.

Reply:
good idea. done.